vignettes/synthetic_news_data.Rmd

```diff
@@ -101,6 +101,97 @@ This dataset is available from the [NHSRDatasets](https://CRAN.R-project.org/pac
 
 For mode information about the [synthpop](http://gradientdescending.com/generating-synthetic-data-sets-with-synthpop-in-r/) package.
 
+## What is NEWS?
+
+NEWS is short for the National Early Warning Score. [NHS England have provided a detailed introduction here](https://www.england.nhs.uk/ourwork/clinical-policy/sepsis/nationalearlywarningscore/)
+
+The latest iteration of the NEWS score is NEWS2
+
+The premise of NEWS is that physiology such as heart rate (pulse), respiration rate,  consciousness (GCS or AVPU) are all routinely measured. 
+GCS = Glasgow Coma Score (Categorical score 3-15) measuring the Eyes, verbal and motor responses.
+AVPU = A categorical description of how concious a patient is A - Alert, V - Responds to voice, P - Responds to painful stimuli, U - Unresponsive
+
+However there are a range of professional groups who use these measurements, and it can e challenging to recognise the deteriorating patient from the raw measurements alone especially if you do not often work with acutely unwell patients
+
+NEWS(2) provides categorical classifications for distinct ranges of physiology. Each category is scored 0-3
+
+The more abnormal a measure of physiology the greater the categorical score attributed. The score is supposed to be calculated at the time the physiology is measured. In a hospital this is often when the nurse or healthcare assistant completes their observation rounds.
+
+The categorical NEWS score then is linked to distinct actions that should be followed. These actions will typically be localised by organisations depending on the level of resource that is available to support medical emergencies.
+
+There are some criticisms of NEWS that were addressed by NEWS2. These were that normal measures of Oxygen saturation (SpO2) were not universal and often meant over escalation of "normal" abnormal physiology in patients with respiratory diseases such as COPD. These were addressed though adjusted ranges for SpO2.
+
+There have also been concerns that in some cases the NEWS score has been introduced to settings (often mandatory) where it has not been validated. The Score was developed by the Royal College of Physicians. They often represent clinical specialties who work in, in-patient medicine. As such the data that was used to develop the score  was based on data from patients who were typically out of the acute phase of their illness and so abnormal physiology was a measure post therapeutic interventions. In most Cases NEWS has been shown to be robust to these criticisms.
+
+NEWS is more work for (typically nursing) staff to complete, NEWS is also not validated as an incomplete score for example where just a heart rate, Blood pressure and SpO2 are recorded which is a common set of measurements in most outpatient settings.
+
+### Here are some code chunks for the calculation of NEWS sub scores:
+
+#### Systolic Blood pressure
+```{r}
+sbp_news <- NEWS_var%>%
+  mutate (sbp = as.numeric(pulse)) %>%
```

Comment:
Should this be `mutate(sbp = as.numeric(syst))` or something similar? In NHSRdatasets we have both pulse and syst and heart rate below also refers to pulse.